src/_includes/layouts/services.njk

```diff
@@ -0,0 +1,51 @@
+---
+layout: layouts/fullwidth.njk
+---
+
+{% from "../../../node_modules/govuk-frontend//dist/govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{{ govukBreadcrumbs({
+  items: [
+    {
+      text: site.name,
+      href: "/"
+    },
+    {
+      text: "Services",
+      href: "/services"
+    },
+    {
+      text: name
+    }
+  ]
+}) | safe }}
+<h1 class="govuk-heading-xl">{{ name }}
+</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l ">{{summary}}</h2>
+    {{ content | safe }} 
+    
```

Comment:
Again a future enhancement, but a diagram of all the service's received/sent events would be helpful